MSY/Day1/짝수와 홀수.js

```diff
@@ -0,0 +1,7 @@
+function solution(num) {
+    if (num%2==0) {
```

Comment:
```suggestion
    if (num % 2 == 0) {
```

여기도 마찬가지로 스페이스바를 넣어주면 가독성이 올라갈 것 같아요!

문제 너무 잘 푸셨습니다!! 고생하셨어요😄